inc/class-ac-yoast-seo-acf-content-analysis.php

```diff
@@ -80,7 +80,11 @@ public function boot() {
 	 * Boots the plugin for dev environment.
 	 */
 	public function boot_dev() {
-		$version = ( -1 === version_compare( get_option( 'acf_version' ), 5 ) ) ? '4' : '5';
+		$registry = Yoast_ACF_Analysis_Facade::get_registry();
+		$configuration = $registry->get( 'config' );
+
+		$version = ( -1 === version_compare( $configuration->get_acf_version(), 5 ) ) ? '4' : '5';
```

Comment:
This shorthand if-statement doesn't seem necessary (and readable).

I'd prefer defaulting to version 4, and only checking if we need to bump to version 5 via `version_compare`.